apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.ts

```diff
@@ -119,6 +148,86 @@ router.post(
           tid_new,
         });
 
+        const serverJob = await editor.prepareServerJob();
+        try {
+          await editor.executeWithServerJob(serverJob);
+          return res.redirect(req.originalUrl);
+        } catch {
+          return res.redirect(res.locals.urlPrefix + '/edit_error/' + serverJob.jobSequenceId);
+        }
+      }
+    } else if (req.body.__action === 'update_assessment') {
+      const infoAssessmentPath = encodePath(
```

Comment:
Why is `encodePath` being used here? This will URL-encode any special characters, and the resulting path definitely won't exist on disk because filesystems don't use URL-encoding.

Reply:
Removed `encodePath` in [ca2ca5d](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/ca2ca5d0e50c8f76d05e06333bbe2e8d3c49cf1c).